framework/src/types.ts

```diff
@@ -145,3 +146,92 @@ export interface ActionInfoForBus {
 	readonly module: string;
 	readonly name: string;
 }
+
+/* Start codec */
+export interface CodecSchema {
+	account: Schema;
+	blockSchema: Schema;
+	blockHeaderSchema: Schema;
+	blockHeadersAssets: {
+		[key: number]: Schema;
+	};
+	baseTransaction: Schema;
+	transactionsAssets: {
+		[key: number]: Schema;
+	};
+}
+
+export interface AccountJSON {
```

Comment:
i don't think these JSON types should be defined in the framework. Also, this type changes depending on what user defines. (it's not static type). same for other cases